src/terms/local.jl

```diff
@@ -90,8 +90,9 @@ function (::AtomicLocal)(basis::PlaneWaveBasis{T}) where {T}
 end
 
 @timing "forces: local" function compute_forces(::TermAtomicLocal,
-                                                basis::PlaneWaveBasis{T},
-                                                ψ, occ; ρ, kwargs...) where {T}
+                                                basis::PlaneWaveBasis,
+                                                ψ, occ; ρ, kwargs...)
+    T = promote_type(eltype(basis), real(eltype(ψ[1])))
```

Comment:
I wonder if we should keep the `where {T}` and then just use the paradigm
```
T = promote_type(T, real(eltype(ψ[1])))
```

Reply:
Well, if we choose to do that, it raises the error
```
LoadError: syntax: local variable name "T" conflicts with a static parameter
```
so maybe rather something like
```
@timing "forces: local" function compute_forces(::TermAtomicLocal, basis::PlaneWaveBasis{TT}, ψ, occ; ρ, kwargs...) where TT  
T = promote_type(TT, real(eltype(ψ[1])))                             

```

Reply:
ok.